front/package.json

```diff
@@ -29,6 +29,8 @@
     "test:ci": "jest --ci --reporters=default --reporters=jest-junit -w 3 --coverage --coverageReporters=text-summary --",
     "cypress:open": "cypress open",
     "cypress:run": "cypress run --spec 'cypress/e2e/**/*.ts' --browser chrome",
+    "playwright": "npx playwright install --with-deps && playwright test",
```

Comment:
`npx playwright install --with-deps` this is needed to ensure the necessary dependencies for the tests to run. There is an alternative way to [set this up](https://playwright.dev/docs/docker) in docker that I haven't tried yet.